modules/services/MapRouletteService.js

```diff
@@ -0,0 +1,314 @@
+    return this._cache.rtree.search(extent.bbox()).map(d => d.data);
+  }
+
+
+  /**
+   * loadTiles
+   * Schedule any data requests needed to cover the current map view
+   */
+  loadTiles() {
+    // determine the needed tiles to cover the view
+    const projection = this.context.projection;
+    const tiles = this._tiler.getTiles(projection).tiles;
+
+    // abort inflight requests that are no longer needed
+    this._abortUnwantedRequests(this._cache, tiles);
+
+    // issue new requests..
+    for (const tile of tiles) {
+      if (this._cache.loadedTile[tile.id] || this._cache.inflightTile[tile.id]) continue;
+
+      const [x, y, z] = tile.xyz;
+      const extent = this.context.systems.map.extent();
+      const bbox = extent.bbox();
+
+      const urlBboxSpecifier = `${bbox.minX}/${bbox.minY}/${bbox.maxX}/${bbox.maxY}`;
+
+      const url = `${MAPROULETTE_API}/tasks/box/` + urlBboxSpecifier;
+
+      const controller = new AbortController();
+      this._cache.inflightTile[tile.id] = controller;
+
+      fetch(url, { signal: controller.signal })
+        .then(utilFetchResponse)
+        .then(data => {
+          this._cache.loadedTile[tile.id] = true;
+
+          for (const task of (data ?? [])) {
+            // MapRoulette tasks are uniquely identified by an
+            // `id`
+              const id = task.id;
+              let loc = [task.point.lng, task.point.lat];
+              loc = this._preventCoincident(loc);
+
+              let d = new Task(loc, this, id, { task });
+
+              this._cache.tasks.set(d.id, d);
+              this._cache.rtree.insert(this._encodeIssueRtree(d));
+          }
+
+          this.context.deferredRedraw();
+          this.emit('loadedData');
+        })
+        .catch(err => {
+          if (err.name === 'AbortError') return;    // ok
+          this._cache.loadedTile[tile.id] = true;   // don't retry
+        })
+        .finally(() => {
+          delete this._cache.inflightTile[tile.id];
+        });
+    }
+  }
+
+
+  /**
+   * loadTaskDetailAsync
+   * @param   task
+   * @return  Promise
+   */
+  loadTaskDetailAsync(task) {
+    // Issue details only need to be fetched once
+    if (task.elems !== undefined) return Promise.resolve(task);
+
+    const localeCode = this.context.systems.l10n.localeCode();
+    const url = `${MAPROULETTE_API}/task/${task.id}?langs=${localeCode}`;
+    const handleResponse = (data) => {
+      // Associated elements used for highlighting
+      // Assign directly for immediate use in the callback
+      task.details = data;
+      this.replaceTask(task);
+    };
+
+    return fetch(url)
+      .then(utilFetchResponse)
+      .then(handleResponse)
+      .then(() => task);
+  }
+
+
+  /**
+   * getIcon
+   * Get the default icon to use
+   * @return  icon name
+   */
+  getIcon() {
+    return 'maki-circle-stroked';
+  }
+
+
+  /**
+   * postUpdate
+   * @param   issue
+   * @param   callback
+   */
+  postUpdate(issue, callback) {
```

Comment:
This method is almost completely unchanged from the Osmose service- we'll need a different API call to post that a task is complete. 

We may need separate methods for lock/unlock, and 'mark as complete'.